akka-http-core/src/test/scala/akka/http/scaladsl/settings/SettingsEqualitySpec.scala

```diff
@@ -11,18 +11,7 @@ import org.scalatest.WordSpec
 
 class SettingsEqualitySpec extends WordSpec with Matchers {
 
-  val config = ConfigFactory.parseString("""
-    akka.http.routing {
-      verbose-error-messages = off
-      file-get-conditional = on
-      render-vanity-footer = yes
-      range-coalescing-threshold = 80
-      range-count-limit = 16
-      decode-max-bytes-per-chunk = 1m
-      decode-max-size = 8m
-      file-io-dispatcher = ${akka.stream.blocking-io-dispatcher}
-    }
```

Comment:
this is the best example that that class was on the wrong place. 

In this test we check for the settings equality. The test works perfectly for all settings except for the RoutingSettings. To make it work, we need to define the settings in place because they are not in the reference.conf here.